docs/plugins/virtual_environment_docs/4-developer-tools.md

```diff
@@ -25,12 +25,6 @@ These _check_ your code.
 These _auto-modify_ your code.
 * [black](https://github.com/psf/black)
   Forces code to follow specific style, indentations, etc...
```

Comment:
Ah right -I only see one rule though

https://github.com/napari/napari/blob/b9dc8dca1e1afa7567586e7ffa7a36cc7cd9fb6f/pyproject.toml#L245

so I'll again just point people to the pyproject.toml in case there are future updates.

Reply:
yes. The list of used linters is `.pre-commit-config.yaml`  https://github.com/napari/napari/blob/b9dc8dca1e1afa7567586e7ffa7a36cc7cd9fb6f/.pre-commit-config.yaml#L6

Reply:
@melissawm If you scroll down from your link, ruff is performing all of the task that you had in the original text since it is a drop-in replacement for these tools. Perhaps link out to ruff docs: https://docs.astral.sh/ruff/linter/#the-ruff-linter